content/src/content/jcr_root/apps/core/wcm/components/modal/v1/modal/_cq_htmlTag/.content.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    cq:tagName="div"
```

Comment:
we don't add this html tag for other components, recommend we move these css classes internally.